amundsen_application/static/js/config/config-default.ts

```diff
@@ -24,6 +24,7 @@ const configDefault: AppConfig = {
   },
   indexDashboards: {
     enabled: false,
+    isBeta: false,
```

Comment:
Changes in `config` could be removed if we opt for all the previous suggestions to check the resourceType in certain places to show the `Flag`, and hardcoding it on `DashboardPage` as well. Overall I favor that option because this isn't a permanent Amundsen feature, and it will be nice to introduce the least amount of temp code.